src/AdoNet/Orleans.Persistence.AdoNet/MySQL-Persistence.sql

```diff
@@ -51,15 +51,13 @@ CREATE TABLE OrleansStorage
     -- The mapping is done in the code. The
     -- *String columns contain the corresponding clear name fields.
     --
-    -- If there are duplicates, they are resolved by using GrainIdN0,
-    -- GrainIdN1, GrainIdExtensionString and GrainTypeString fields.
+    -- If there are duplicates, they are resolved by using GrainIKey,
```

Comment:
I believe this should read "GrainIdKey"
```suggestion
    -- If there are duplicates, they are resolved by using GrainIdKey,
```